lib/src/modules/login/widgets/code_text_field.dart

```diff
@@ -35,7 +35,7 @@ class EnterCodeTextField extends StatelessWidget {
             ),
             floatingLabelBehavior: FloatingLabelBehavior.always,
           ),
-          autovalidate: autoValidate,
+          autovalidateMode: (autoValidate == true) ? AutovalidateMode.always : AutovalidateMode.disabled,
```

Comment:
autoValidate => returns bool
autoValidate == true => returns bool

no need of == true